packages/service/src/domain/songs/songs-service.ts

```diff
@@ -15,7 +15,11 @@ class SongsService {
   }
 
   async getAll(): Promise<Song[]> {
-    return this.repository.findAllOrderByTimesPlayedDesc();
+    return this.repository.getOrderedByTimesPlayedDesc(0);
+  }
+
+  async getTop(limit: number): Promise<Song[]> {
```

Comment:
```suggestion
  async getWithHighestPlayCount(limit: number): Promise<Song[]> {
```
albo coś podobnego. `getTop` jest IMO zbyt lakoniczne.